code/index.html

```diff
@@ -2,11 +2,35 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project Name</title>
-    <!-- dont forget the link to the css file -->
-    <!-- dont forget the meta tag for responsiveness -->
+    <title>Kajsas form</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://kit.fontawesome.com/18157e8d5f.js" crossorigin="anonymous"></script>
   </head>
+  
   <body>
-    <!-- Build your site up as you want it! Make sure to use the form element and try to send that form to httpbin by adding action="https://httpbin.org/anything" and method="POST" to your form tag. -->
+    <div class="theFormBorder border-xs-none" >
+      <div class="theForm">
+        <h2>Sign up</h2>
+        <form action="http://httpbin.org/anything" method="POST">
+        <span class="fas fa-envelope icon"></span>
+        <input type="email" id="email" name="email" value="kajsa@kajsa.se" required><br>
+        <span class="fas fa-lock icon"></span>
+        <input type="password" id="password" class="password" name="password" value="Your password" required><br>
```

Comment:
The HTML-tag <br> really comes in handy - but it's not used very much anymore. Another tip to try out is to wrap the icon and the input in a [label](https://www.w3schools.com/tags/tag_label.asp) to make them go on the same row:
```
<label>
  <span class="fas fa-lock icon"></span>
  <input
    type="password"
    id="password"
    class="password"
    name="password"
    value="Your password"
    required
  />
</label>
```